### PR TITLE
[Keyboard] Add handwired F411

### PR DESCRIPTION
**Pull request by riblee**
## Description

Added a new Blackpill F411 based handwired keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/riblee_f411/keymaps/default/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# The default Preonic layout - largely based on the Planck's
```
Comment (fauxpark) [inline keyboards/handwired/riblee_f411/keymaps/default/readme.md:1]: This file can be deleted.

File: keyboards/handwired/riblee_f411/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Handwired Preonic
```
Comment (fauxpark) [inline keyboards/handwired/riblee_f411/README.md:1]: Please rename this file to lowercase.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/riblee_f411/keymaps/default/readme.md (hunk 1 above)
Comment (riblee) [reply keyboards/handwired/riblee_f411/keymaps/default/readme.md]: Done

File: keyboards/handwired/riblee_f411/README.md (hunk 2 above)
Comment (riblee) [reply keyboards/handwired/riblee_f411/README.md]: Done

**Comment by riblee** [comment]
Could somebody merge this PR please? I have some work going on which is depends on this.

File: keyboards/handwired/riblee_f411/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+enum preonic_layers {
+    _QWERTY,
+    _COLEMAK,
+    _DVORAK,
+    _LOWER,
+    _RAISE,
+    _ADJUST
+};
+
+enum preonic_keycodes {
```
Comment (fauxpark) [inline keyboards/handwired/riblee_f411/keymaps/default/keymap.c:29]:
```suggestion
enum custom_keycodes {
```

File: keyboards/handwired/riblee_f411/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum preonic_layers {
```
Comment (fauxpark) [inline keyboards/handwired/riblee_f411/keymaps/default/keymap.c:20]:
```suggestion
enum layer_names {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Just these two minor things, LGTM otherwise.

File: keyboards/handwired/riblee_f411/keymaps/default/keymap.c (hunk 3 above)
Comment (riblee) [reply keyboards/handwired/riblee_f411/keymaps/default/keymap.c]: I've changed them

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!
(Just waiting on Travis now)

**Comment by riblee** [comment]
Thank you!

**Pull request merged**